package.json

```diff
@@ -14,7 +15,9 @@
     "clean-es": "rimraf es",
     "build-es": "npm run clean-es && builder run build-babel -- -d es",
     "watch-es": "watch \"npm run build-es\" src/ -d",
-    "build": "builder concurrent --buffer build-lib build-es",
+    "clean-types": "rimraf types",
+    "build-types": "npm run clean-types && tsc -p ./tsconfig.dts.json",
```

Comment:
Does this still work if the command is instead:

```js
 "build-types": "npm run clean-types && tsc -p tsconfig.dts.json",
```

(The `./` is problematic for windows cmd users IIRC)

Reply:
it should, trying locally

Reply:
As a Windows user, it wasn't problematic for me, but you are the Johnny Cash of your dot-slash.

Reply:
@DanielRosenwasser -- I didn't actually check your user profile until Ken told me -- and it's fair to say that any of your command suggestions should be good to go cross-platform-wise unequivocally 😄 

I must say that

> Johnny Cash of your dot-slash

is truly a priceless gem that I hope enters our lexicon with a lot more regularity 😆